R/create-project.R

```diff
@@ -93,7 +93,7 @@ create_directories <- function(
 }
 
 write_core_files <- function(data) {
-  core_files <- list.files(fs::path_package("PEMr", "templates", "core"))
+  core_files <- list.files(fs::path_package("PEMr", "templates", "core"), pattern = c("*.qmd|*.R"))
```

Comment:
This is fine, but shouldn't really need it because shouldn't be rendering in this folder. But probably worth adding the rendering artifacts to `.gitignore` and `.Rbuildignore`

Reply:
Yeah, I added it as it was throwing an error with the .qmd base file. I think however we should stick with the .R file and keep the .qmd/Rmd for the vignette so I can change it back once that is confirmed.